bootstrap.sh

```diff
@@ -42,8 +48,19 @@ do
       export SINCE=$(echo "$2" | sed "s/:/%3A/g")
       shift
       ;;
+    "--fhir")
+      export FHIR_VERSION=$2
+      shift
+      ;;
   esac
   shift
 done
 
+if [ "${FHIR_VERSION}" == "R4" ]
```

Comment:
Ok for now but if we ever add a 3rd version, this should be converted to a case statement.